### PR TITLE
EE-7419: Provide a way to enable statistics without monitoring

### PR DESCRIPTION
**Pull request by fsgonz**

### CODE REVIEW

File: src/main/java/org/mule/runtime/api/util/MuleSystemProperties.java [hunk 1]
```diff
@@ -172,6 +172,20 @@
    */
   public static final String SHARE_ERROR_TYPE_REPOSITORY_PROPERTY = SYSTEM_PROPERTY_PREFIX + "share.errorTypeRepository";
 
+  /**
+   * When enabled this System Property, the statistics are enabled even if the monitoring service is not acivated.
```
Comment (elrodro83) [inline src/main/java/org/mule/runtime/api/util/MuleSystemProperties.java:176]: explain that this property is only read at app deployment time

File: src/main/java/org/mule/runtime/api/util/MuleSystemProperties.java [hunk 2]
```diff
@@ -172,6 +172,20 @@
...
+   * 
+   * @since 4.4, 4.3.1
+   */
+  public static final String MULE_ENABLE_STATISTICS = SYSTEM_PROPERTY_PREFIX + "enable.statistics";
+
+  /**
+   * When enabled this System Property, the payload statistics are disabled independently of the statistics flag.
```
Comment (elrodro83) [inline src/main/java/org/mule/runtime/api/util/MuleSystemProperties.java:183]: explain that this property is only read at app deployment time

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Comment by fsgonz** [comment]
--validate

**Pull request merged**